scripts/build.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * This is a little wrapper around `tsdx build` that helps to rename some files
+ * since TSDX has some unexpected behavior around file naming. This behavior
```

Comment:
is this script necessary? it is not in the other ones

Reply:
Good catch, it's not. It's a leftover from an old attempt at doing the build. I'll remove it.